analyzers/tests/SonarAnalyzer.UnitTest/TestCases/LossOfFractionInDivision.CSharp10.cs

```diff
@@ -1,10 +1,63 @@
-﻿public struct S
+﻿using System;
+using System.Threading.Tasks;
+
+public struct S
 {
     public decimal Property { get; set; }
 
     public void M()
     {
-        (Property, var _) = (3 / 2, // FN
-                             3 / 2);
+        (Property, var _) = (3 / 2, 3 / 2);    // Noncompliant
```

Comment:
can you please add the message and the position on this line? (not on all, but at least on one sample, to ensure message + position make sense)